packages/bundle-size/src/utils/sortComparedReport.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @param {import('../utils/compareResultsInReports').ComparedReportEntry} a
+ * @param {import('../utils/compareResultsInReports').ComparedReportEntry} b
+ */
+function compareReports(a, b) {
+  return a.packageName.localeCompare(b.packageName) || a.path.localeCompare(b.path);
+}
+
+/**
+ * @param {import('../utils/compareResultsInReports').ComparedReport} report
+ */
+module.exports = function sortComparedReport(report) {
```

Comment:
```suggestion
module.exports = function sortReportsByPackageName(report) {
```

Reply:
I think that current naming is better as we could change sorting behavior later, I don't want to couple it with fields. Do you have other suggestions for naming? 🤔

Reply:
in that case perhaps comment the sort function with the behaviour, because sorting based on the large entry object isn't very clear until you have to look at the comparator

Reply:
@ling1726 I added a comment in 86bb6b8.